src/askui_runner/modules/agents/containers.py

```diff
@@ -0,0 +1,34 @@
+from functools import cached_property
+from typing import Dict
+
+from ..core.infrastructure.agent.agent_file_service import AskUIAgentFileService
+from ..core.infrastructure.askui import AskUiAccessToken
+from ..core.infrastructure.files.askui import AskUiFilesService
+
+
+from .models import AgentsConfig
+
+
+class SyncContainer:
+    def __init__(self, config: AgentsConfig):
+        self._config: AgentsConfig = config
+
+    @cached_property
+    def _access_token(self) -> AskUiAccessToken:
+        return AskUiAccessToken(access_token=self._config.credentials.access_token)
+
+    @cached_property
+    def _base_http_headers(self) -> Dict[str, str]:
+        return {"Authorization": self._access_token.to_auth_header()}
+
+    @cached_property
+    def _agent_file_service(self) -> AskUIAgentFileService:
```

Comment:
This one is accessed outside the module so it should not be private
```suggestion
    def agent_file_service(self) -> AskUIAgentFileService:
```